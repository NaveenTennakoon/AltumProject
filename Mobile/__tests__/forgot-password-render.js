import React from 'react';
import FORGOT from '../screens/ForgotPassword';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<FORGOT />).toJSON();
  expect(tree).toMatchSnapshot();
});