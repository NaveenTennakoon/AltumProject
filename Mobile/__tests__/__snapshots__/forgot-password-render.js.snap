// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  behaviour="padding"
  onLayout={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#DCDCDC",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "uri": "https://cdn2.f-cdn.com/contestentries/68791/9261050/5337f7fab2996_thumb900.jpg",
      }
    }
    style={
      Object {
        "flex": 1,
        "height": "100%",
        "justifyContent": "center",
        "position": "absolute",
        "width": "100%",
      }
    }
  />
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 18,
        "marginBottom": 30,
        "marginHorizontal": 30,
        "textAlign": "center",
        "textShadowColor": "rgba(0, 0, 0, 0.75)",
        "textShadowOffset": Object {
          "height": 1,
          "width": -1,
        },
        "textShadowRadius": 10,
      }
    }
  >
    We get it. Things do happen. Have you forgot your password? Enter the email address below and we will email a password reset link
  </Text>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderBottomColor": "#F5FCFF",
        "borderBottomWidth": 1,
        "borderRadius": 30,
        "elevation": 5,
        "flexDirection": "row",
        "height": 50,
        "marginBottom": 20,
        "shadowColor": "#808080",
        "shadowOffset": Object {
          "height": 2,
          "width": 0,
        },
        "shadowOpacity": 0.25,
        "shadowRadius": 3.84,
        "width": 340,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      placeholder="Email Address"
      rejectResponderTermination={true}
      returnKeyType="go"
      style={
        Object {
          "borderBottomColor": "#FFFFFF",
          "flex": 1,
          "height": 45,
          "marginLeft": 16,
        }
      }
      underlineColorAndroid="transparent"
    />
    <Image
      source={
        Object {
          "uri": "https://img.icons8.com/nolan/64/000000/filled-message.png",
        }
      }
      style={
        Object {
          "height": 30,
          "justifyContent": "center",
          "marginRight": 15,
          "width": 30,
        }
      }
    />
  </View>
  <View
    accessible={true}
    clickable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#000000",
        "borderRadius": 30,
        "elevation": 19,
        "flexDirection": "row",
        "height": 45,
        "justifyContent": "center",
        "marginBottom": 20,
        "opacity": 1,
        "shadowColor": "#808080",
        "shadowOffset": Object {
          "height": 9,
          "width": 0,
        },
        "shadowOpacity": 0.5,
        "shadowRadius": 12.35,
        "width": 340,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 16,
          "fontWeight": "700",
        }
      }
    >
      Submit Request
    </Text>
  </View>
  <View
    accessible={true}
    clickable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "flex-start",
        "backgroundColor": "transparent",
        "flexDirection": "row",
        "height": 15,
        "justifyContent": "flex-start",
        "marginBottom": 10,
        "opacity": 1,
        "width": 300,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 15,
        }
      }
    >
      back to Login
    </Text>
  </View>
</View>
`;
